packages/create-plugin/templates/common/.config/webpack/utils.ts

```diff
@@ -50,14 +50,17 @@ export async function getEntries(): Promise<Record<string, string>> {
     })
   );
 
-  return plugins.reduce((result, modules) => {
-    return modules.reduce((result, module) => {
-      const pluginPath = path.dirname(module);
-      const pluginName = path.relative(process.cwd(), pluginPath).replace(/src\/?/i, '');
-      const entryName = pluginName === '' ? 'module' : `${pluginName}/module`;
+  return plugins.reduce(
+    (result, modules) => {
+      return modules.reduce((innerResult, module) => {
+        const pluginPath = path.dirname(module);
+        const pluginName = path.relative(process.cwd(), pluginPath).replace(/src\/?/i, '');
+        const entryName = pluginName === '' ? 'module' : `${pluginName}/module`;
 
-      result[entryName] = module;
-      return result;
-    }, result);
-  }, {});
+        innerResult[entryName] = module;
+        return innerResult;
+      }, result);
+    },
+    {} as Record<string, string>
```

Comment:
Did this part generate a typescript error as well when you tried to import the base webpack config? I think it's better with the change, as it's inferring more, only asking to make sure I understand correctly.